websites/ubuntu.lt/deploy.yml

```diff
@@ -132,18 +129,18 @@
   # Let's Encrypt
 
   - name: "let's encrypt: apt packages"
-    apt: pkg={{ item }} state=latest
-    with_items:
-    - libaugeas0
-    - libssl-dev
-    - libffi-dev
-    - ca-certificates
+    apt:
+      name:
+      - libaugeas0
+      - libssl-dev
+      - libffi-dev
+      - ca-certificates
 
   - name: "let's encrypt: pip packages"
     pip: name={{ item }} virtualenv=/opt/letsencrypt state=latest
     with_items:
-    - letsencrypt
-    - letsencrypt-apache
+    - certbot
+    - certbot-apache
```

Comment:
Maybe it is wort installing certbot from apt?

https://packages.ubuntu.com/focal/certbot

Reply:
Personally I would prefer if these playbooks didn't touch certbot at all.

I have certbot managed from our POV playbooks.  Getting a new cert is one line in a hostvars/servername file + one ansible-playbook run.

Reply:
In that case, we can probably remove certbot from here and just leave a comment that they are managed somewhere else.